crates/compiler/gen_llvm/src/llvm/build.rs

```diff
@@ -6770,10 +6750,14 @@ pub fn to_cc_return<'a>(
     layout: InLayout<'a>,
 ) -> CCReturn {
     let return_size = layout_interner.stack_size(layout);
-    let pass_result_by_pointer = match env.target_info.operating_system {
-        roc_target::OperatingSystem::Windows => return_size > env.target_info.ptr_width() as u32,
-        roc_target::OperatingSystem::Unix => return_size > 2 * env.target_info.ptr_width() as u32,
-        roc_target::OperatingSystem::Wasi => return_size > 2 * env.target_info.ptr_width() as u32,
+    // TODO: loop back and update this. It actually cares about the full abi (arch + os)
```

Comment:
Just checking - the plan is to update this in a future PR?

Reply:
Yeah, I know that this is wrong in some cases, but need to dig into each abi to figure out the edge cases.